client/pages/emailData.js

```diff
@@ -0,0 +1,29 @@
+const emailData = [
+  {
+    num: 1,
+    to: "curtis@gmail.com",
+    subject: "Assignment Extension",
+  },
+  {
+    num: 2,
+    to: "seang1833@gmail.com",
+    subject: "Meeting at 2pm",
+  },
+  {
+    num: 3,
+    to: "service@amazon.ca",
+    subject: "Delayed Delivery",
+  },
+  {
+    num: 4,
+    to: "terry@gmail.com",
+    subject: "Project Status",
+  },
+  {
+    num: 5,
+    to: "student@example.ca",
+    subject: "TEST",
+  },
+];
+
+export default emailData;
```

Comment:
These email datas are very different from the "real ones" you will receive from the backend. Is this template data needed?